tests/test_models.py

```diff
@@ -1,28 +1,28 @@
-import pytest
-from votekit.models import Outcome
+# import pytest
```

Comment:
if these tests aren't meant to pass they should either be removed or marked with pytest.mark.skip
https://docs.pytest.org/en/7.1.x/how-to/skipping.html

Reply:
uncommented, tests are passing but thanks for the note!